src/MAUI/Maui.Samples/CategoryListPage.xaml

```diff
@@ -1,44 +1,91 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
-<ContentPage x:Class="ArcGIS.CategoryListPage"
-             xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
-             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml">
+<ContentPage
+    x:Class="ArcGIS.CategoryListPage"
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:converters="clr-namespace:ArcGIS.Converters">
+
+    <ContentPage.Resources>
+        <converters:SampleToBitmapConverter x:Key="SampleToBitmapConverter" />
+    </ContentPage.Resources>
     <ContentPage.ToolbarItems>
```

Comment:
Titling this page "ArcGIS Maps SDK for .NET (MAUI)" maybe via a label could be a nice addition.